lib/workerpool.go

```diff
@@ -727,8 +727,19 @@ func (pool *WorkerPool) RacMaint(racReq racAct) {
 	pool.poolCond.L.Unlock()
 	// TODO: C++ worker logs one event for each worker, in the worker, so
 	// we keep the same. Think about changing it
-	for i := 0; i < cnt; i++ {
+	for i := 0; i < len(racMaintWorkers); i++ {
+		poolModName, _ := racMaintWorkers[i][0].(string)
+		shardId, _ := racMaintWorkers[i][1].(int64)
+		workerId, _ := racMaintWorkers[i][2].(int64)
```

Comment:
Can we also track pid?

Reply:
added pid to CAL event

E16:18:21.36	RAC_ID	0	0	poolModName=hera-test&workerId=1&pid=83228&shardId=0&tm=1714992488&exitTime=1714992541&exitInSec=40sec
E16:18:21.36	DB_UNAME	 MyDB	0	
E16:18:21.36	RAC_ID	0	0	poolModName=hera-test&workerId=2&pid=83227&shardId=0&tm=1714992488&exitTime=1714992581&exitInSec=80sec
E16:18:21.37	DB_UNAME	 MyDB	0	
E16:18:21.37	RAC_ID	0	0	poolModName=hera-test&workerId=3&pid=83232&shardId=0&tm=1714992488&exitTime=1714992621&exitInSec=120sec
E16:18:21.37	DB_UNAME	 MyDB	0	
E16:18:21.37	RAC_ID	0	0	poolModName=hera-test&workerId=4&pid=83231&shardId=0&tm=1714992488&exitTime=1714992661&exitInSec=160sec
E16:18:21.37	DB_UNAME	 MyDB	0	
E16:18:21.37	RAC_ID	0	0	poolModName=hera-test&workerId=5&pid=83229&shardId=0&tm=1714992488&exitTime=1714992701&exitInSec=200sec